### ISSUE TITLE
Log file as requested by RD 2.0.8

### ISSUE THREAD

**Issue by eteichm** [rubberduck-vba/Rubberduck#2175]
RD asked to save my work and restart. 
Here is the Log file as requested by RD 2.0.8. 
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/398904/RubberduckLog.txt)

**Comment by Vogel612** [comment]
There's a **lot** of resolver errors in here, but they usually don't crash things. In addition there's a NullReferenceException:

```
2016-08-03 09:49:35.0269;ERROR;DynamicInjector457a0cb3b02941cfa60ce7c047ec1ce3;System.NullReferenceException: Object reference not set to an instance of an object.
   at Rubberduck.UI.Command.AddTestMethodCommand.<CanExecuteImpl>b__6_2(Declaration a) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\UI\Command\AddTestMethodCommand.cs:line 58
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at Rubberduck.UI.Command.AddTestMethodCommand.CanExecuteImpl(Object parameter) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\UI\Command\AddTestMethodCommand.cs:line 57
   at Rubberduck.UI.Command.CommandBase.CanExecute(Object parameter) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\UI\Command\CommandBase.cs:line 36;
2016-08-03 09:49:35.0599;ERROR;DynamicInjector11cf453a0aa14878a63fbb942ee201d0;System.NullReferenceException: Object reference not set to an instance of an object.
   at Rubberduck.UI.Command.AddTestMethodExpectedErrorCommand.<CanExecuteImpl>b__6_2(Declaration a) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\UI\Command\AddTestMethodExpectedErrorCommand.cs:line 63
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at Rubberduck.UI.Command.AddTestMethodExpectedErrorCommand.CanExecuteImpl(Object parameter) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\UI\Command\AddTestMethodExpectedErrorCommand.cs:line 62
   at Rubberduck.UI.Command.CommandBase.CanExecute(Object parameter) in C:\Users\hosch\Documents\Visual Studio 2015\Projects\Rubberduck\RetailCoder.VBE\UI\Command\CommandBase.cs:line 36;
```

This was already mentioned in #2204 though. I assume these are the same thing ..

**Comment by retailcoder** [comment]
Deemed fixed as of 2.0.9.
